src/main/java/com/palpal/dealightbe/domain/item/application/ItemService.java

```diff
@@ -92,26 +92,29 @@ public ItemsRes findAllForMember(double xCoordinate, double yCoordinate, String
 		return ItemsRes.from(items);
 	}
 
-//	public ItemRes update(Long itemId, ItemReq itemReq, Long memberId) {
-//		Store store = storeRepository.findByMemberId(memberId)
-//			.orElseThrow(() -> {
-//				log.warn("GET:READ:NOT_FOUND_STORE_BY_MEMBER_ID : {}", memberId);
-//				return new EntityNotFoundException(NOT_FOUND_STORE);
-//			});
-//
-//		checkDuplicatedItemName(itemReq.name(), store.getId());
-//
-//		Item item = itemRepository.findById(itemId)
-//			.orElseThrow(() -> {
-//				log.warn("GET:READ:NOT_FOUND_ITEM_BY_ID : {}", itemId);
-//				return new EntityNotFoundException(NOT_FOUND_ITEM);
-//			});
-//
-//		Item updatedItem = ItemReq.toItem(itemReq, store);
-//		item.update(updatedItem);
-//
-//		return ItemRes.from(item);
-//	}
+	public ItemRes update(Long itemId, ItemReq itemReq, Long memberId, ImageUploadReq imageUploadReq) {
+		Store store = storeRepository.findByMemberId(memberId)
+			.orElseThrow(() -> {
+				log.warn("GET:READ:NOT_FOUND_STORE_BY_MEMBER_ID : {}", memberId);
+				return new EntityNotFoundException(NOT_FOUND_STORE);
+			});
+
+		Item item = itemRepository.findById(itemId)
+			.orElseThrow(() -> {
+				log.warn("GET:READ:NOT_FOUND_ITEM_BY_ID : {}", itemId);
+				return new EntityNotFoundException(NOT_FOUND_ITEM);
+			});
```

Comment:
반복되는 부분이니 private 메소드로 분리해줘도 좋을 것 같아요!

```suggestion
		Store store = getStore(memberId);
		Item item = getItem(itemId);
```